packages/powersync_core/lib/powersync_core.dart

```diff
@@ -9,7 +9,6 @@ export 'src/database/powersync_database.dart';
 export 'src/exceptions.dart';
 export 'src/log.dart';
 export 'src/open_factory.dart';
-export 'src/powersync_database.dart';
```

Comment:
This file just exports `database/powersync_database.dart`, which is already exported here.